.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - 'stable'
+  - '4.4'
+
+install: npm install
```

Comment:
`install` and `script` steps are redundant.
- The language `node_js` has a [default test script](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Default-Test-Script).
- The same applies to how the dependencies are installed. https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Travis-CI-uses-npm

Could you please remove them?